Exercises-2-Basic/src/exercises/Num7.java

```diff
@@ -0,0 +1,29 @@
+package exercises;
+
+/*
+ Write a Java program to print numbers between 1 to 100 which are divisible by 3, 5 and by both. 
+ */
+public class Num7 {
+
+    public static void main(String[] args) {
+	System.out.println("Divided by 3: ");
+	for (int i = 1; i < 100; i++) {
+	    if (i % 3 == 0)
```

Comment:
The body of the if statement should be inside of {}.
There are a couple of places in this file.

Reply:
Can you give me an example?

Reply:
i added more lines :)